runtime/queries/cpp/highlights.scm

```diff
@@ -12,13 +12,6 @@
 (template_method
   name: (field_identifier) @function)
 
-(template_function
-  name: (identifier) @function)
-
-(function_declarator
-  declarator: (qualified_identifier
-    name: (identifier) @function))
```

Comment:
Without this stanza, some function declarations are highlighted as properties instead:

```cpp
class C : D {
  void efg() {
  }
}